jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -142,6 +142,15 @@ def search(self, error_on_fail=False):
                                       query_region_result['dec'],
                                       unit='deg')
 
+        elif self.catalog_selected == 'Gaia':
+            from astroquery.gaia import Gaia
+
+            Gaia.ROW_LIMIT = 5_000
```

Comment:
```suggestion
            Gaia.ROW_LIMIT = 1_000
```
We'll need to make this configurable in the plugin.